app/models/video.rb

```diff
@@ -13,4 +13,8 @@ def self.search_by_title(search_term)
     where('title LIKE ?', "%#{search_term}%").order(created_at: :desc)
   end
  
+  def rating
+    reviews.average(:rating).round(1) if reviews.average(:rating)
```

Comment:
Nice job pushing the work to the database. You are the first person I have seen use this here!